inputs/select.vue

```diff
@@ -135,57 +161,48 @@
       }
     },
     methods: {
-      update(option) {
-        if (_.isArray(option)) {
-          // set all existing options in data to false
-          const newData = _.mapValues(_.cloneDeep(this.data), () => false);
-
-          // for each of the selected options, set the related key in the data to true
-          _.forEach(option, (o) => newData[o.value] = true);
-
-          this.$store.commit(UPDATE_FORMDATA, { path: this.name, data: newData });
-        } else if (_.isObject(option)) {
-          // single new checked option
-          this.$store.commit(UPDATE_FORMDATA, { path: this.name, data: option.value });
+      // input may be stored as simply a value (scalar) or the entire input object
+      formatOptionForStore (o) {
+        if (_.isObject(o) && !this.args.storeRawData) {
+          return o[this.keys.value]
+        } else {
+          return o
         }
       },
+      // basically, all input takes the object form
+      formatOptionForInput (o) {
+        // start-case labels for scalar options
+        if (_.isString(o) || _.isNumber(o)) {
```

Comment:
Probably not super useful but to be consistent with Keen I added support for number values, e.g. 
```
options: [1, 2, 3]
```